components/user/userDetails/userActivities/UserActivities.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Stars from '../../../stars/Stars';
+
+interface UserActivitiesProps {
+  login: string;
+  photo?: string;
+}
+
+const UserActivities: React.FC<UserActivitiesProps> = ({ login, photo }) => {
+  return (
+    <section className="w-full md:w-1/3 mt-8 md:mt-0">
+      <h3 className="font-bold text-center md:hidden">Dotychczasowa aktywność</h3>
+      <div className="flex items-center justify-between px-2 mt-4">
```

Comment:
pewnie warto wyciągnąć z tego komponent Activity, no ale rozumiem, że to jest tylko dummy przykład narazie

Reply:
tak - potem będzie to zmapowane. Zrobione jest tylko, żeby jakoś style dobrać.